src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -66,10 +73,45 @@ class DropdownAutoCompleteMenu extends React.Component {
      * Dropdown menu alignment.
      */
     alignMenu: PropTypes.oneOf(['left', 'right']),
+
     /**
      * Should menu visually lock to a direction (so we don't display a rounded corner)
      */
     blendCorner: PropTypes.bool,
+
+    /**
+     * Max height of dropdown menu
+     */
+    maxHeight: PropTypes.number,
+
+    /**
+     * Add z-index to menu. Note this is not added by default because some
+     * implementations of DropdownAutocomplete DEPEND on no z-index.
+     * This is when we want to blend the bottom border of the actor button with the dropdown menu
+     * so that it looks a bit more seamless.
+     */
+    zIndex: PropTypes.number,
+
+    /**
+     * Search input's placeholder text
+     */
+    searchPlaceholder: PropTypes.string,
+
+    /**
+     * Padding value for search input (requires the `padding` shortcut value)
+     */
+    searchPadding: PropTypes.string,
```

Comment:
these two props worry me. Can you explain why we need them? IMO if it just looks weird or too different we should:

A. Make it look better  / moar the same everywhere
B. Pass a styled child component
C. Figure out some kind of semantically named state to pass in that controls the way it looks. 

Ideally we can avoid passing around individual css lines as props because it enables nearly infinite component states and makes maintenance really challenging.

Reply:
I agree. Option A would be ideal but I think it's a bit too idealistic. Option B makes it difficult to re-use the good set of default components we have (and makes it a bit too easy to break design consistency). I think individual css rules as a prop is a good balance between full CSS control (e.g. `searchStyle`) and customizability.

I do like option C though, maybe `*Size`?